test/server/cards/01_SOR/units/PatrollingVWing.spec.js

```diff
@@ -0,0 +1,20 @@
+describe('Patrolling V-Wing', function () {
+    integration(function () {
+        describe('Patrolling V-Wing\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['patrolling-vwing'],
+                    },
+                    player2: {}
+                });
+            });
+
+            it('should draw', function () {
+                this.player1.clickCard(this.patrollingVwing);
+                expect(this.player1.hand.length).toBe(1);
```

Comment:
Let's add a check that the other player has not drawn a card, just for completeness